config/locales/en/errors.yml

```diff
@@ -457,7 +457,7 @@ en:
           attributes:
             pays_council_tax:
               inclusion: Select yes if your client pays Council Tax where they usually live
-            council_tax_amount:
+            amount:
```

Comment:
add `not_a_number: How much they pay yearly must be a number, like 50`


As per https://docs.google.com/spreadsheets/d/1gdDpuyqWO6MJgUT1SOy0nbHd8AAdp1YebBzh1WDG040/edit#gid=0&range=G82